apps/workspaces/tasks.py

```
@@ -241,6 +241,29 @@ def post_to_integration_settings(workspace_id: int, active: bool):
+    payload = {
+        'tpa_name': 'Fyle Netsuite Integration'
+    }
+
+    if errors is not None:
+        payload['errors_count'] = errors
+
+    if is_token_expired is not None:
+        payload['is_token_expired'] = is_token_expired
+        
+    try:
+        patch_request(url, payload, refresh_token)
```

Ruff (0.11.9)

Undefined name `patch_request`

(F821)